Logs: Update dataplane specification to have labels field instead of attributes (#37)
grafana/dataplane: docs/contract/logs.md

- **ID field** - _optional_
  - The first string field with the name `id` is the id field.
  - Unique identifier of the log line
- **Labels field** - _optional_

>   - The first field with the name `labels` is the labels field.

Author: I removed  "string field" as Labels field is `other` or `json raw message` field. Not a string field.

>   - This field represent additional labels of the log line.

Reviewer: i recommend adjusting this to not say `labels` in the explanation, because that's very loki/grafana specific.. maybe something like:
```
- This field represents additional information about the log line.
```
WDYT?

Author: ```suggestion
  - This field represent additional information about the log line.
```

Author: Good point

  - This field represent additional attributes of the log line.
- **Labels field** - _optional_
  - The first field with the name `labels` is the labels field.
  - This field represent additional labels of the log line.
  - Field type must be json raw message type. Example value: `{}`, `{"hello":"world", "foo": 123.45, "bar" :["yellow","red"], "baz" : { "name": "alice" }}`

>     - Should not be empty string.

Author: I might be missing something, but `labels` is not a string field, so I don't understand this point and therefore I removed it. Let me know if I just misunderstood something and we should keep this.
